src/plugins/BasebandRIL.js

```diff
@@ -153,7 +153,7 @@ module.exports = class BasebandRIL extends OverlayPlugin {
             this.simOperatorMMC.placeholder = 'eg: 20814';
             this.simOperatorMMC.pattern = '[0-9]{5,6}';
             this.simOperatorMMC.title = 'SIM Operator MCC/MNC';
-            this.simOperatorMMC.addEventListener('keyup', this.checkSimImsiErrors.bind(this));
+            this.instance.addListener(this.simOperatorMMC, 'keyup', this.checkSimImsiErrors.bind(this));
```

Comment:
if checkSimImsiErrors become an arrow fn (const checkSimImsiErrors = ()=>{}) you will be able to write
`this.instance.addListener(this.simOperatorMMC, 'keyup', this.checkSimImsiErrors);
`

Reply:
honestly in this case I think it would be more difficult to read especially since this function has a big doc comment above it